hilti/toolchain/include/ast/node.h

```diff
@@ -240,7 +240,15 @@ class Node {
     virtual ~Node();
 
     /** Returns the node tag associated with the instance's class. */
-    auto nodeTag() const { return _node_tags.back(); }
+    node::Tag nodeTag() const {
+        // Get the last non-zero tag. The last element(s) may be unset
```

Comment:
This surprises me, when did you see unset tags at the end?

Reply:
since `Tags` [is an array](https://github.com/zeek/spicy/blob/b1e305b92975bf9d1ab52049fc5c7472526fc1f2/hilti/toolchain/include/ast/node-tag.h#L17), anything that had less than 2 other bases would just return `0`. Which turns out to be most nodes.

Reply:
Oh, I see: it's an array *and* the method isn't used anywhere currently, meaning it wasn't obvious that it was broken. :-)  May have been a `vector` originally and the method is just a left-over.